src/main/java/com/keeper/homepage/domain/merit/dao/MeritLogRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.keeper.homepage.domain.merit.dao;
+
+import com.keeper.homepage.domain.merit.entity.MeritLog;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface MeritLogRepository extends JpaRepository<MeritLog, Long> {
+  Page<MeritLog> findByGiver_Id(Pageable pageable, Long memberId);
```

Comment:
참고만 해주세용~

둘 다 동작하지만, 저는 find로 가져오는게 단건일땐 `find`, 복수일땐 `findAll`을 써주는 편입니다~

```suggestion
  Page<MeritLog> findAllByGiverId(Pageable pageable, Long memberId);
```